### ISSUE TITLE
Entities Support

### ISSUE THREAD

**Issue by Valkirie** [jajo-11/Schematic_to_Java_Python_3#3]
Would be great if the software could support Entities (painting is the one i'm looking for).

**Comment by jajo-11** [comment]
I agree, but sadly I have little to no time to maintain this project nowadays. So I suggest you look elsewhere or implement your own schematic/nbt parser.
